internal/envconfig/envconfig.go

```diff
@@ -43,8 +43,7 @@ var (
 	// EnforceALPNEnabled is set if TLS connections to servers with ALPN disabled
 	// should be rejected. The HTTP/2 protocol requires ALPN to be enabled, this
 	// option is present for backward compatibility. This option may be overridden
-	// by setting the environment variable "GRPC_ENFORCE_ALPN_ENABLED" to "true"
-	// or "false".
+	// by setting the environment variable "GRPC_ENFORCE_ALPN_ENABLED" to "false".
```

Comment:
I suspect you didn't want these diffs.

Reply:
Reverted. I initially added an env var to toggle happy eyeballs, which was later removed it. I made a similar change to the doc comment for new env var based on reviewer feedback.